entities/statistics.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Statistics
+  include Validation
+
+  def top_reader_stats(orders, quantity)
+    orders.group_by(&:reader).sort_by { |_key, value| -value.count }.first(quantity).to_h.keys
```

Comment:
you can use Array#reverse instead of using minus for **value.count**, the same for most_popular_books_stats